internal/reader/home_chain.go

```diff
@@ -223,18 +223,35 @@ func (r *homeChainPoller) GetOCRConfigs(
 	ctx context.Context, donID uint32, pluginType uint8,
 ) ([]OCR3ConfigWithMeta, error) {
 	var ocrConfigs []OCR3ConfigWithMeta
+	var allConfigs GetAllConfigs
+
 	err := r.homeChainReader.GetLatestValue(
 		ctx,
 		r.ccipConfigBoundContract.ReadIdentifier(consts.MethodNameGetOCRConfig),
 		primitives.Unconfirmed,
 		map[string]any{
 			"donId":      donID,
 			"pluginType": pluginType,
-		}, &ocrConfigs)
+		}, &allConfigs)
 	if err != nil {
 		return nil, fmt.Errorf("error fetching OCR configs: %w", err)
 	}
 
+	r.lggr.Infow(
+		"GetOCRConfigs",
+		"activeConfig", allConfigs.ActiveConfig,
+		"candidateConfig", allConfigs.CandidateConfig,
+	)
+
+	// Not empty
```

Comment:
Unnecessary comment. General comment guidelines: don't say _what_ the code is doing; if there is some obscure reasoning behind what the code is doing then include that in the comment.